docs/user-guides/configurations.md

```diff
@@ -115,3 +115,9 @@ If using Iceberg Format，please refer to [Iceberg configurations](https://icebe
 | ---------------------------------- | ---------------- | ----------------------------------       |
 | base.hive.auto-sync-schema-change  | true             | Whether synchronize schema changes of Hive Table from HMS             |
 | base.hive.auto-sync-data-write     | false            | Whether synchronize data changes of Hive Table from HMS, this should be true when writing to Hive    |
+
+### Dashboard configurations
+
+| Key                            | Default | Description                                                                                                                  |
+|--------------------------------|---------|------------------------------------------------------------------------------------------------------------------------------|
+| dashboard.thread-count         | 20      | The number of threads used for dashboard to read metadata. Currently only using for paimon's 'Files' and 'Transactions' page |
```

Comment:
I don't think this is a good configuration key name.

`dashboard.mainfest.io.thread-count` ?

Reply:
yeah

Reply:
@zhoujinsong WDYT ?